tests/10-EdvClient.spec.js

```diff
@@ -1,6 +1,8 @@
 /*!
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
+'use strict';
```

Comment:
no use scrict in ESM (where import is used).  Applies throughout.  We DO want use strict on CommonJS code that uses require.

Reply:
Updated it.